CRM/Financial/Form/FinancialBatch.php

```diff
@@ -83,8 +83,7 @@ public function preProcess() {
   public function buildQuickForm() {
     parent::buildQuickForm();
     $this->setPageTitle(ts('Financial Batch'));
-
-    if (isset($this->_id)) {
+    if (!empty($this->_id)) {
```

Comment:
this is fine - although I think it would be better to define the property on the class